tests/integration/targets/meraki_ms_l3_interface/tasks/main.yml

```diff
@@ -3,59 +3,64 @@
 
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 ---
-- block:
+- name: Block of tests
+  block:
   - name: Create network with type switch
-    meraki_network:
+    cisco.meraki.meraki_network:
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkSwitch
       type: switch
       timezone: America/Chicago
     delegate_to: localhost
     register: create_net_switch
 
   - name: Claim a device into a network
-    meraki_device:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
+    cisco.meraki.meraki_device:
+      auth_key: '{{ auth_key }}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkSwitch
       serial: '{{ serial_switch_l3 }}'
       state: present
     delegate_to: localhost
 
   - name: Query all l3 interfaces
-    meraki_ms_l3_interface:
+    cisco.meraki.meraki_ms_l3_interface:
       auth_key: '{{ auth_key }}'
       state: query
       serial: '{{ serial_switch_l3 }}'
     delegate_to: localhost
     register: query_all
 
-  - set_fact:
+  - name: Set fact interface_id_1
+    ansible.builtin.set_fact:
       interface_id_1: '{{query_all.data.1.interface_id}}'
-    ignore_errors: true
+    failed_when: false
```

Comment:
Bringing this up like on the others. What is the purpose of having these as I'm afraid they can cover problems?